### PR TITLE
Refactor Flow files

### PR DESCRIPTION
**Pull request by yurishkuro**

### CODE REVIEW

**Comment by yurishkuro** [comment]
@black-adder

File: src/_flow/metrics.js [hunk 1]
```diff
@@ -19,6 +19,26 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+// Counter tracks the number of times an event has occurred
+declare interface Counter {
+    // Add adds the given value to the counter.
```
Comment (black-adder) [inline src/_flow/metrics.js]: should this be increment?

File: src/_flow/samlper-thrift.js [hunk 2]
```diff
@@ -81,15 +27,24 @@ declare type RateLimitingSamplingStrategy = {
...
+// PerOperationSamplingStrategies defines a collection of sampling strategies
+// per operation name, and a pair of parameters for the default sampling strategy
+// applicable to unknown operation names.
+declare type PerOperationSamplingStrategies = {
+    defaultSamplingProbability: number,
+    defaultLowerBoundTracesPerSecond: number,
+    perOperationStrategies: Array<OperationSamplingStrategy>
+}
+
 declare type SamplingStrategyResponse = {
```
Comment (black-adder) [inline src/_flow/samlper-thrift.js]: PerOperationSamplingStrategies needs to be added here. I guess that'll be part of a separate PR?

File: src/_flow/tchannel.js [hunk 3]
```diff
@@ -19,8 +19,9 @@
...
 
-// Timer tracks how long an envent has occurred.
-declare interface Timer {
-    // Records the time passed in.
-    record(value: number): void;
-}
+declare type TChannelSpan = {
+    id: Array<number>,
+    traceid: Array<number>,
+    parentid: Array<number>,
```
Comment (black-adder) [inline src/_flow/tchannel.js]: shouldnt this guy be optional?

File: src/_flow/metrics.js (hunk 1 above)
Comment (yurishkuro) [inline src/_flow/metrics.js]: fixed
